lightning/src/ln/channelmanager.rs

```diff
@@ -5549,6 +5549,7 @@ where
 		}
 		let preimage_update = ChannelMonitorUpdate {
 			update_id: CLOSED_CHANNEL_UPDATE_ID,
+			counterparty_node_id: None,
```

Comment:
It seems like we may know the counterparty node id here, any reason we're leaving it unset?